README.md

```diff
@@ -251,45 +309,40 @@ Describing ACH file 'test/testdata/ppd-debit.ach'
   1           1           1                  100000000         0
 ```
 
-## Getting Started
-
-- [Running ACH Server](./docs/README.md#running-moov-ach-server)
-- [Intro to ACH](./docs/intro.md)
-- [Create an ACH File](./docs/create-file.md)
-
-## Guides
+### In-Browser ACH File Parser
+Using our [in-browser utility](http://oss.moov.io/ach/), you can instantly convert ACH files into JSON. Either paste in ACH file content directly or choose a file from your local machine. This tool is particulary useful if you're handling sensitive PII or want perform some quick tests, as operations are fully client-side with nothing stored in memory. We plan to support bidirectional conversion in the near future.
 
-- [ACH File Structure](./docs/file-structure.md)
-- [Balanced offset files](./docs/balanced-offset.md)
-- [Merging ACH files](./docs/merging-files.md)
-- [ACH Server metrics](./docs/metrics.md)
+### SDKs
 
-### From Source
-
-This project uses [Go Modules](https://github.com/golang/go/wiki/Modules) and Go 1.14 or higher. See [Golang's install instructions](https://golang.org/doc/install) for help in setting up Go. You can download the source code and we offer [tagged and released versions](https://github.com/moov-io/ach/releases/latest) as well. We highly recommend you use a tagged release for production.
-
-```
-$ git@github.com:moov-io/ach.git
+Below are some SDKs generated from the API documentation:
 
-# Pull down into the Go Module cache
-$ go get -u github.com/moov-io/ach
+- [Node SDK](https://www.npmjs.com/package/ach-node-sdk) | [GitHub](https://github.com/moov-io/ach-node-sdk)
 
-$ go doc github.com/moov-io/ach BatchHeader
-```
+- TODO - OpenAPI Guide
 
-### Configuration
+## Learn About ACH
 
-| Environmental Variable | Description | Default |
-|-----|-----|-----|
-| `ACH_FILE_TTL` | Time to live (TTL) for `*ach.File` objects stored in the in-memory repository. | 0 = No TTL / Never delete files (Example: `240m`) |
-| `LOG_FORMAT` | Format for logging lines to be written as. | Options: `json`, `plain` - Default: `plain` |
-| `HTTP_BIND_ADDRESS` | Address for paygate to bind its HTTP server on. This overrides the command-line flag `-http.addr`. | Default: `:8080` |
-| `HTTP_ADMIN_BIND_ADDRESS` | Address for paygate to bind its admin HTTP server on. This overrides the command-line flag `-admin.addr`. | Default: `:9090` |
-| `HTTPS_CERT_FILE` | Filepath containing a certificate (or intermediate chain) to be served by the HTTP server. Requires all traffic be over secure HTTP. | Empty |
-| `HTTPS_KEY_FILE`  | Filepath of a private key matching the leaf certificate from `HTTPS_CERT_FILE`. | Empty |
+- [Intro to ACH](./docs/intro.md)
+- [Create an ACH File](./docs/create-file.md)
+- [ACH File Structure](./docs/file-structure.md)
+- [Balanced Offset Files](./docs/balanced-offset.md)
+- [Merging ACH Files](./docs/merging-files.md)
+- [ACH Server Metrics](./docs/metrics.md)
 
+## FAQ
```

Comment:
![image](https://user-images.githubusercontent.com/29054071/104524089-ac1caf00-55b3-11eb-8261-d03762d1d51e.png)
Dang, it looks like the answer is hidden by default and you have to click the arrow to unhide. Could we make that unhidden?

Reply:
Sure thing, will fix